consent/helpers.py

```diff
@@ -71,10 +81,18 @@ def get_course_data_sharing_consent(username, course_id, enterprise_customer_uui
         enterprise_customer__uuid=enterprise_customer_uuid
     )
 
+
 def get_program_data_sharing_consent(username, program_id, enterprise_customer_uuid):
+    """
+    Get the data sharing consent object associated with a certain user of a customer for a program.
+
+    :param username: The user that grants consent.
+    :param program_id The program for which consent is granted.
```

Comment:
Nit: missing ":" at the end of program_id. With that said, maybe rename to `program_uuid` everywhere as we changed what it's also referred to as in the view? If that's too much work, nevermind.